.travis.yml

```diff
@@ -1,4 +1,3 @@
-sudo: false
```

Comment:
The reason I switch to apt-get is that the addons stuff you tried just didn't work. I tried it a few different ways with no luck. However, I have another solution, we can just install linkchecker from PyPI!